src/ci/docker/scripts/ubuntu16-apt-get.sh

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Mind renaming this to include the term "cross" in the script name? I think this list of packages is mostly suitable for building crosstool-ng and such and running that inside the container.